project/plugins.sbt

```diff
@@ -3,4 +3,22 @@ addSbtPlugin("com.github.sbt"    % "sbt-release"   % "1.1.0")
 addSbtPlugin("org.scoverage"     % "sbt-scoverage" % "1.6.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"    % "5.2.0")
 addSbtPlugin("com.eed3si9n"      % "sbt-assembly"  % "0.15.0")
-addSbtPlugin("com.github.sbt"    % "sbt-jacoco"    % "3.4.0")
+
+// sbt-jacoco - workaround related dependencies required to download
+lazy val ow2Version = "9.5"
+lazy val jacocoVersion = "0.8.9-absa.1"
+
+def jacocoUrl(artifactName: String): String = s"https://github.com/AbsaOSS/jacoco/releases/download/$jacocoVersion/org.jacoco.$artifactName-$jacocoVersion.jar"
```

Comment:
Are there plans to release the plugin or contribute back to the original?

Reply:
There is started contribution to the original. Sbt-jacoco is using the origin, so it will take some time to get it here.